R/metrics.R

```diff
@@ -568,3 +574,13 @@ eval_safely <- function(expr, expr_nm, data = NULL, env = caller_env()) {
     }
   )
 }
+
+
+call_remove_duplicate_arguments <- function(call) {
```

Comment:
All the work happens in this function.

It looks at the arguments that were sets in the function created by `tweak_metric()` and if there are any (like `na_rm` or `estimator` it will zap them from the call object. This way they don't get used twice